src/api/v1/paths/doughs/{doughId}.js

```diff
@@ -19,10 +20,24 @@ const get = async (req, res) => {
 };
 
 /**
- * stub of patch function
+ * Update a dough with the ID specified in the request body
  *
- * @returns null
+ * @type {RequestHandler}
  */
-const patch = async () => null;
+const patch = async (req, res) => {
+  try {
+    if (req.params.doughId !== req.body.data.id) {
+      return errorBuilder(
+        res,
+        '409',
+        `ID ${req.params.doughId} in URL parameter does not match ID ${req.body.data.id} in body`,
```

Comment:
Not important, but I think `req.body.data.id` and `req.params.doughId` can be destructured in the first place and be reused.

Reply:
That's true, although looking at it more I do like how as it is it's very clear where each id value comes from, and it might be less clear after destructuring. I don't feel super strongly about that though.